app/src/components/ViewEvent/ViewEvent.tsx

```diff
@@ -48,7 +50,7 @@ export const ViewEvent = ({
       <WavySubHeader eventId={eventId}>
         <div className={style.headerContainer}>
           <h1 className={style.header}>{event.title}</h1>
-          {userCanEdit && (
+          {userCanEdit && eventId && (
```

Comment:
`const isNotCreatingEvent = eventId !== undefined`
`userCanEdit && isNotCreatingEvent && ....`

Er noe jeg foretrekker for å tydeligere si hva det betyr